client/app/expenses/page.tsx

```diff
@@ -25,9 +25,9 @@ export default function ExpensePage() {
         }
       />
       {isExpensePerTime ? (
-        <ExpenseTrackingPerDate />
+        <ExpenseTrackingPerDate key='per-date' />
       ) : (
-        <ExpenseTrackingPerMonth />
+        <ExpenseTrackingPerMonth key='per-month' />
```

Comment:
Can I ask you what the key is for?

Reply:
Sorry I forgot to delete it during try and error!
https://github.com/HAK2024/fairy-share/pull/119/commits/28be3529e8f9947aeb11a0bb8380197f30450260